DStarPlanner:
    goal_tol: .1